tests/unit/Makefile.inc

```diff
@@ -95,3 +95,4 @@ unit1608_CPPFLAGS = $(AM_CPPFLAGS)
 
 unit1609_SOURCES = unit1609.c $(UNITFILES)
 unit1609_CPPFLAGS = $(AM_CPPFLAGS)
+
```

Comment:
Minor nit: An unnecessary empty line slipped in here.

Reply:
oops